charts/jaeger/values.yaml

```diff
@@ -2,6 +2,66 @@
 # This is a YAML-formatted file.
 # Jaeger values are grouped by component. Cassandra values override subchart values
 
+v2:
```

Comment:
Let's not try to make this branch support both v1 and v2. We already have a chart which support v1, let's focus 100% on v2 from now on.